packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -242,9 +242,8 @@ export interface IZoweUSSTreeNode extends IZoweTreeNode {
     saveSearch?(ussFileProvider: IZoweTree<IZoweUSSTreeNode>);
     /**
      * uploads selected uss node(s) to from clipboard to mainframe
-     *
      */
-    copyUssFile?();
+    pasteUssTree?();
```

Comment:
yes we need to keep both for now here and mark the old as deprecated incase extenders are using it for something. Since we use these tree apis like any extender would I think us changing our call in Zowe Explorer to use the new would be ok. What do you think @zFernand0 ?